chsdi/templates/index.mako

```diff
@@ -40,7 +40,7 @@
           <a href="checker">Checker for home page</a> <br>
           <a href="checker_dev">Checker for dev page</a> <br> 
       <h2 id="qrcode">QRCode service</h2>
-          <a href="qrcodegenerator?url=http://s.geo.admin.ch/e83c57af1">QRCode service</a> <br>
+          <a href="qrcodegenerator?url=https://s.geo.admin.ch/e83c57af1">QRCode service</a> <br>
```

Comment:
I think we should, since theses are .geo.admin.ch links

https://mf-chsdi3.dev.bgdi.ch/ltpoa/shorten/6435fdca66